lang/cs/Org.Apache.REEF.Tests/Functional/Failure/EvaluatorWithRunningTaskDelayedPoisonTest.cs

```diff
@@ -127,7 +114,7 @@ public void OnNext(IFailedEvaluator value)
 
                 Logger.Log(Level.Info, "Received all expected failed Tasks.");
 
-                const string expectedStr = "expected a single Context with Context ID " + ContextId + ".";
+                var expectedStr = "expected a single Context with Context ID " + ContextId + ".";
 
                 if (value.FailedContexts == null)
```

Comment:
Yes, of course the check will vary from test to test. These two tests have identical checks because in both of them failure is guaranteed to happen after task started running => context is guaranteed to be created.